app/assets/javascripts/questions.js

```diff
@@ -26,7 +26,23 @@ function initEditQuestion() {
   $('#question_text, #question_answers_attributes_0_text').keyup(validateEditQuestionInput);
 
   // track qset user created a question in
-  $('#question_qset_id').change(function() {
+  $('#qset-id-input').change(function() {
     Cookies.set('new_question_qset_id', this.value, { expires: 1000 });
   });
+
+  selectQsetForNewQuestion();
+
+}
+
+// Closes modal, displays qset name next to Choose Qset button, and populates hidden qset-id input
+function selectQsetForNewQuestion() {
+  $('.qset-selectable').click(function() {
+    $('.qset-display').text($(this).find('.qset-name-modal').text().trim());
+    $('.qset-display').removeClass("no-qset-chosen");
+    $('qset-display').addClass("qset-chosen");
```

Comment:
Looks like we're missing the CSS class qualifier here, the period `.` at the start of `qset-display`? Also, Jquery methods always conveniently return the object they acted on, so they can be chained, like so:
```
$('.qset-display').removeClass("no-qset-chosen").addClass("qset-chosen");
```
...that said, looking at the `.qset-chosen` style def in the CSS, we probably don't need it at all, since the defaults for these values are effectively doing the same thing (the default font color isn't straight `black`, but it's close enough, and by leaving it default we make it blend better and make it conform with any tweaks to app-wide defaults at a later time).